### PR TITLE
Dep 2763

### PR DESCRIPTION
**Pull request by tonymyers**
# DEP-(StoryNumberHere)

## Checklist PR Raiser
 - [x]  I've ensured code coverage has not decreased
 - [x]  I've dealt with any new compilation warnings
 - [x]  I've ensured the team's coding standards have been met (TBC)?
 - [ ]  If relevant, I've created corresponding app-config-XXX changes?
 - [x]  If I've created new test data, I've ensured there is no real person's data (see https://confluence.tools.tax.service.gov.uk/display/CD/Test+Data+in+the+Open) 
 - [x]  I've considered the impact my changes have on the UI/Journey tests (please do not break them)
-  [ ]  If relevant, I've added links to associated PRs

## Checklist PR Reviewer
 - [x]  I've checked to ensure all relevant unit tests have been written
 - [x]  I've checked to ensure the team's coding standards have been met
 - [x]  I've executed the acceptance test pack locally to ensure there are no functional regressions (Performance tests where relevant)
 - [x]  If I merge I will ensure I use squash and merge

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -35,6 +36,7 @@
   "dependencies": {
     "gulp-cli": "^2.3.0",
     "jquery": "^3.5.1",
+    "jsdom": "^19.0.0",
```
Comment (lmugnai) [inline package.json:39]: think we can remove this too

File: test/uk/gov/hmrc/digitalengagementplatformskin/javascripts/controllers/ProactiveChatController.spec.js [hunk 2]
```diff
@@ -19,28 +43,7 @@ describe("ProactiveChatController", () => {
...
-
-    it("appends chat transcript div to page when some div id is found on page", () => {
-        const commonChatController = new CommonChatController();
-        let chatContainer = document.createElement("div");
-        chatContainer.setAttribute("id", "HMRC_CIAPI_Fixed_1");
-        document.body.appendChild(chatContainer);
-
-        commonChatController._showChat();
-
-        expect(document.getElementById("ciapiSkinChatTranscript").innerHTML).not.toBe(null);
```
Comment (lmugnai) [inline test/uk/gov/hmrc/digitalengagementplatformskin/javascripts/controllers/ProactiveChatController.spec.js:43]: has this really all been removed or are these tests still in place?

Comment (tonymyers) [reply test/uk/gov/hmrc/digitalengagementplatformskin/javascripts/controllers/ProactiveChatController.spec.js]: Added back in, well spotted

File: test/uk/gov/hmrc/digitalengagementplatformskin/javascripts/utils/ClickToChatButtons.spec.js [hunk 3]
```diff
@@ -33,6 +33,7 @@ function c2cObj(displayState, launchable = false) {
 
 describe("ClickToChatButtons", () => {
     it("adds a button with active state", () => {
+        console.error = jest.fn();
```
[the remark this answers is not in the document]
Comment (tonymyers) [reply test/uk/gov/hmrc/digitalengagementplatformskin/javascripts/utils/ClickToChatButtons.spec.js]: Removed

**Pull request merged**